source/Appccelerate.EventBroker.Specs/Scoping/ScopeSpecifications.cs

```diff
@@ -4,13 +4,13 @@
 //
 //   Licensed under the Apache License, Version 2.0 (the "License");
 //   you may not use this file except in compliance with the License.
-//   You may obtain a copy of the License at
+//   You may obtain a copy of the the License at

```

Comment:
The header is not well formed

Reply:
Your're realy picky, there was just one of them 👍 :)
I fixed it and expanded the time span of the copyright to be up to date.

I tested the new version with our project and it worked like a charm.